src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies?: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = (props) => {
+  const { movies = [] } = props;
+
+  return (
+    <div className="movies">
+      {movies.map(movie => (
+        <div className="card" key={movie.imdbId}>
+          <MovieCard film={movie} />
+        </div>
+      ))}
+    </div>
```

Comment:
isn't it supposed to be a list? like a 'ul' kind of thing

Reply:
I think that all movies can be bem-blocks, so maybe will be better to use <section> for outer tag and <article> for each movie instead of simple div.
And in <ul> all inner elements looks like bem-elements, so I decided not to use them